### PR TITLE
nuka cola in vending machines is now in a bottle (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
https://github.com/HippieStation/HippieStation/pull/12606 port of this pr
nuka cola is now in a bottle
#### Changelog

:cl:  
rscadd: bottled nuka cola
/:cl:

### CODE REVIEW

**Review by JamieD1 — APPROVED** [review]

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -426,7 +426,14 @@
 	icon_state = "trappistbottle"
 	volume = 50
 	list_reagents = list(/datum/reagent/consumable/ethanol/trappist = 50)
-
+	
+// hippie start -- New bottled version of Nuka Cola for use in Cola vending machines
+/obj/item/reagent_containers/food/drinks/bottle/nukacola
+	name = "Nuka Cola"
+	desc = "Don't cry, Don't raise your eye, It's only nuclear wasteland."
```
Comment (Ktlwjec1) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: **d**on't raise your eye, **i**t's

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 2]
```diff
@@ -426,7 +426,14 @@
...
 	list_reagents = list(/datum/reagent/consumable/ethanol/trappist = 50)
-
+	
+// hippie start -- New bottled version of Nuka Cola for use in Cola vending machines
+/obj/item/reagent_containers/food/drinks/bottle/nukacola
+	name = "Nuka Cola"
+	desc = "Don't cry, Don't raise your eye, it's only nuclear wasteland."
+	icon_state = "nuka_colaglass"
+	list_reagents = list(/datum/reagent/consumable/nuka_cola = 50)
+// hippie end
```
Comment (alexkar598) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: Remove the hippie start and hippie end, its not a matter of giving credit or not, its just that we dont need those markers

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm (hunk 2 above)
Comment (Hopekz) [reply code/modules/food_and_drinks/drinks/drinks/bottle.dm]: *gasp

**Comment by boodaliboo** [comment]
if you insist

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**